extensions/wikia/Search/css/WikiaSearch.scss

```diff
@@ -639,3 +641,39 @@ $form-negative-side-padding: $form-side-padding*-1;
 		}
 	}
 }
+
+//Breakpoints related changes
+@if ( $breakpoints-layout-enabled ) {
+	.Search .WikiaSearch {
+		.SearchInput {
+			p {
+				margin: 0;
+			}
+
+			.grid-1.alpha {
+				width: 170px;
+				float: left;
+			}
+
+			@media #{$breakpoint-below-small-plus} {
+				.search-v2-input {
+					width: 280px;
+				}
+			}
+		}
+
+		.search-tabs {
+			float: left;
+			margin: 35px 0 0 0;
+			width: 170px;
+		}
+
+		.results-wrapper {
+			@include width-calc("100% - 500px");
+			@media #{$breakpoint-below-small-plus} {
+				@include width-calc("100% - 180px");
```

Comment:
What are `500px` and `180px` values standing for? Something that has actually such widths? If yes, you could put them in a descriptive variable. If these are just magic numbers, give them a comment or sth.